src/components/TransactionList/TransactionRow/TransactionRowTs.ts

```diff
@@ -140,7 +156,23 @@ export class TransactionRowTs extends Vue {
    * @param transaction 
    */
   public getHeight(): string {
-    return this.view.info?.height.compact().toLocaleString()
+    if(this.transactionStatus === 'partial'){
+      return 'partial'
+    }else{
+      return this.view.info?.height.compact().toLocaleString()
       || this.$t('unconfirmed').toString()
+    }
+  }
+  public get transactionStatus() {
+    if (this.transaction.isConfirmed()) {
+      return 'confirmed'
+    } else {
+      if (this.partialTransactions.length > 0 &&
+         this.partialTransactions.some((item) => item.transactionInfo.hash === this.transaction.transactionInfo.hash)) {
```

Comment:
I would prefer a `@Prop({default: false}) isPartial: boolean` inside TransactionRow

and up the latter, TransactionList can use `partialTransactions`